framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -63,4 +91,99 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 			await this._sidechainAPIClient.disconnect();
 		}
 	}
+	public async getCertificateFromAggregateCommit(
+		aggregateCommit: AggregateCommit,
+	): Promise<Certificate> {
+		const block = await this._mainchainAPIClient.invoke<Block>('chain_getBlockByHeight', {
```

Comment:
this should be the client to which node is running the plugin which is a sidechain node, so it should be `this._apiClient.invoke`. we can make a condition that if `_sidechainAPIClient` exists then we use `_sidechainAPIClient.invoke`